### DISCUSSION TITLE
Issues with connecting to remote Jupyter server kernel in VS Code

### DISCUSSION THREAD

**Discussion by Zach789** [microsoft/PlanetaryComputer#268] [Q&A]
I want to utilize the Jupyter server kernel to run my code locally but it is unable to connect to it and I receive this output.

![image](https://github.com/microsoft/PlanetaryComputer/assets/54433167/e7413632-d6d3-4a91-b276-907b37904d73)

![image](https://github.com/microsoft/PlanetaryComputer/assets/54433167/1023775e-c83b-4dce-8610-2ac3736a1483)

The steps I did to try and connect:
1. Start Planetary Computer Hub server with _CPU-Python environment_ in my browser
2. Generate Hub token
3. Enter URL of the server along with token _(https://pccompute.westeurope.cloudapp.azure.com/compute/user/102416353@student.swin.edu.au/?token=<TOKEN>)_
4. Enter credentials 
    - username=email
    - password=account pwd

Steps tried to fix the issue:
- Restart Hub server
- Generate new token
- Clearing VS Code cache
- Reinstalling Jupyter extension 
- Switching to pre-release version of Jupyter extension

I had been successful in doing so on occasion when I am on my university network which leads me to suspect an issue with the network configurations of my local machine instead but I don't have the capabilities to troubleshoot that unfortunately.

Any help would be appreciated.

**Comment by TomAugspurger** [comment]
Hrmm, sorry I don't have any extra guidance to give :/ I don't *think* we've changed anything on the Hub authentication side that would have changed things.

**Reply by Zach789** [reply]
No worries, I couldn't get it to work unfortunately so I'll just run it on the hub instead. Thanks for taking the time to reply.
